saleor/graphql/account/tests/test_account.py

```diff
@@ -4466,26 +4466,30 @@ def test_password_change_invalid_new_password(user_api_client, settings):
 
 
 def test_create_address_mutation(
-    staff_api_client, customer_user, permission_manage_users
+    staff_api_client, customer_user, permission_manage_users, graphql_address_data
 ):
+    # given
     query = ADDRESS_CREATE_MUTATION
+    graphql_address_data["city"] = "Dummy"
     user_id = graphene.Node.to_global_id("User", customer_user.id)
-    variables = {"user": user_id, "city": "Dummy", "country": "PL"}
+    variables = {"user": user_id, "address": graphql_address_data}
+    # when
     response = staff_api_client.post_graphql(
         query, variables, permissions=[permission_manage_users]
     )
     content = get_graphql_content(response)
+    # then
     assert content["data"]["addressCreate"]["errors"] == []
     data = content["data"]["addressCreate"]
-    assert data["address"]["city"] == "Dummy"
+    assert data["address"]["city"] == "DUMMY"
     assert data["address"]["country"]["code"] == "PL"
-    address_obj = Address.objects.get(city="Dummy")
+    address_obj = Address.objects.get(city="DUMMY")
     assert address_obj.user_addresses.first() == customer_user
     assert data["user"]["id"] == user_id
 
     customer_user.refresh_from_db()
     for field in ["city", "country"]:
-        assert variables[field].lower() in customer_user.search_document
+        assert variables["address"][field].lower() in customer_user.search_document
```

Comment:
Could you ale fix the name of this test or add more assets?
Currently, this test checks only  `search_document`.